### DISCUSSION TITLE
"TypeError: Failed to Fetch" in ditto explorer?

### DISCUSSION THREAD

**Discussion by mmcc-xx** [eclipse-ditto/ditto#1605] [Q&A] [answered]
Hi All - I started up ditto via docker compose in an Azure VM running Ubuntu 22.04. I cloned the repo and did a "docker-compose up -d" from deployment/docker without changing anything in the docker-compose file.

I was able to create and retrieve fancy-car via curl from the command line of that Azure VM. I can retrieve fancy-car via curl from my local PC as well.

I set up the Azure network security group to allow access to port 8080 from the internet and am able to access the "You have started Eclipse Ditto" page from a browser, and I'm able to open http://casedittotest.eastus.cloudapp.azure.com:8080/ui/. However after a few seconds I get an error at the bottom right of the page that says "TypeError: Failed to Fetch". All of the tables are empty - no fancy-car is listed.

Any suggestions for how to fix this?

**Comment by thjaeckle** [comment, marked as answer]
Hi @mmcc-xx 
You will have to configure your Ditto environment in the "Environments" section.
By default, the Ditto Explorer UI tries to locate Ditto on "localhost:8080". The UI does not have an own backend, but connects directly to the Ditto HTTP API.

So simply adjust/configure your cloud environment and select it from the dropdown box at the top right.

**Reply by mmcc-xx** [reply]
Perfect! That did the trick. Thanks
